src/components/CloudIcon/index.jsx

```diff
@@ -2,16 +2,18 @@ import React from 'react'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import './styles.css'
 
-const CloudIcon = props => {
+import CloudIcon from "cozy-ui/transpiled/react/Icons/Cloud";
+
+const PasswordsCloudIcon = props => {
   return (
     <Icon
-      icon="cloud"
+      icon={CloudIcon}
       size={16}
       color="var(--charcoalGrey)"
```

Comment:
Should we use semantic var? ;)

Reply:
Do we have a codemod to check the use of this kind of variable and replace it with primary/secondaryColor ?

Reply:
We do not have a semantic variable for charcoalGrey I am afraid :/

Reply:
I will not do that in this PR ;)